src/tabset/tabset.spec.ts

```diff
@@ -213,24 +216,26 @@ describe('ngb-tabset', () => {
   });
 
 
-  it('should change active tab on tab title click', () => {
-    const fixture = createTestComponent(`
+  it('should change active tab on tab title click', fakeAsync(() => {
```

Comment:
Why this change? We are not testing for tabs content here and the test passes even without `fakeAsync`.

Reply:
I just added the fakeAsync / flush to every test that clicked headers, I guess it can be removed on this one.